### ISSUE TITLE
How to use a network B to obtain a tensor and use it to replace the weight of a layer of network A, and this back propagation process will train A and B

### ISSUE THREAD

**Issue by syiswell** [pytorch/pytorch#35877]
### How to use a network B to obtain a tensor and use it to replace the weight of a layer of network A and this backpropagation process will train A and B。

I make a code, which uses the weight of a layer of network A as input into network B. then B output a tensor and I use it to replace the weight of a layer of the network A. But I find the weight only needs the type of nn.Parameter() and I convert the tensor to Parameter type, but I find the weight of network B does not get an update. Can you help me please!  

It's noted that I want to train the weight of network B by the loss of the network A.
- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by jerryzh168** [comment]
please post questions to https://discuss.pytorch.org/ Thanks

**Comment by syiswell** [comment]
thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jerry Zhang"<notifications@github.com&gt;; 
发送时间: 2020年4月7日(星期二) 凌晨1:07
收件人: "pytorch/pytorch"<pytorch@noreply.github.com&gt;; 
抄送: "孙洋"<915142649@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [pytorch/pytorch] How to use a network B to obtain a tensor and use it to replace the weight of a layer of network A, and this back propagation process will train A and B (#35877)





 
Closed #35877.
